packages/amplify-headless-interface/schemas/api/1/UpdateApiRequest.schema.json

```diff
@@ -142,8 +142,7 @@
                 }
             },
             "required": [
-                "cognitoUserPoolId",
-                "mode"
+                "cognitoUserPoolId"
```

Comment:
was this change generated by `yarn generateSchemas`? Seems like mode should still be required and cognitoUserPoolId should no longer be required

Reply:
Yep that was a mistake I made. My bad